bgpd/bgp_attr.c

```diff
@@ -2340,10 +2330,12 @@ bgp_attr_parse_ret_t bgp_attr_prefix_sid(struct bgp_attr_parser_args *args,
 						  args->total);
 		}
 
-		ret = bgp_attr_psid_sub(type, length, args, mp_update);
+		ret = bgp_attr_psid_sub(type, length, args);
 
 		if (ret != BGP_ATTR_PARSE_PROCEED)
 			return ret;
+
+		psid_attr_length -= length + headersz;
```

Comment:
Isn't this subject to underflow unless we compare the length from the wire against the known attribute length before doing this subtraction?

Reply:
Thank you for your review.
Yes, this looks my mistake. thank you so much.
I'll fix that tomorrow. :)

Reply:
I fixed on 38774fc5e6b8da326bbe3700d7d54498aedb154a